nmdc_server/ingest/kegg.py

```diff
@@ -66,8 +143,25 @@ def ingest_ko_module_map(db: Session) -> None:
         )
         db.commit()
 
+    with open(COG_TERM_DEFS) as fd:
```

Comment:
```suggestion
    cogterms = Path(__file__).parent / "data" / "cog" / "cog-20.def.tab"
    with open(cogterms) as fd:
```

this is the docker culprit